.circleci/config.yml

```diff
@@ -118,7 +118,7 @@ jobs:
         default: golang
       go-test-flags:
         type: string
-        default: "-v --tags=debug -timeout 4m"
+        default: "-v --tags=debug -timeout 15m"
```

Comment:
Do we still need this 15m timeout?

Reply:
It certainly needs to be >4m, I will adjust with 2min slack and check how long these tests take (since we have more integration tests now, and we'll be adding retrieval soon).

Reply:
Successful run took 12min. so will leave this at 15min. for now.